test/unit/foreman_google/google_compute_adapter_test.rb

```diff
@@ -6,21 +6,6 @@ module ForemanGoogle
   class GoogleComputeAdapterTest < GoogleTestCase
     subject { ForemanGoogle::GoogleComputeAdapter.new(auth_json_string: gauth_json) }
 
-    describe 'authentication' do
```

Comment:
Not sure why this is failing, or why we need to test it this way, but created redmine for extending test coverage https://projects.theforeman.org/issues/35875